src/ngx_http_lua_timer.c

```diff
@@ -16,6 +16,9 @@
 #include "ngx_http_lua_probe.h"
 
 
+#define MAX_DELAY 2147483648 /* 2 ^ 31 */
```

Comment:
I'm not sure if this threshold makes sense since `ngx_msec_t` is defined as `ngx_uint_t`, which is a 64-bit integer on 64-bit systems, no?